### ISSUE TITLE
Error in graph with lambda

### ISSUE THREAD

**Issue by CJ-Wright** [mrocklin/streamz#76]
It seems that lambda functions now make the graph plotting rather unhappy.
```python
def test_create_file():
    source1 = Stream(stream_name='source1')
    source2 = Stream(stream_name='source2')

    n1 = source1.zip(source2)
    n2 = n1.map(add).scan(mul).map(lambda x: x + 1)
    n2.sink(source1.emit)

    with tmpfile(extension='png') as fn:
        visualize(n1, filename=fn)
        assert os.path.exists(fn)

    with tmpfile(extension='svg') as fn:
        n1.visualize(filename=fn, rankdir="LR")
        assert os.path.exists(fn)

    with tmpfile(extension='dot') as fn:
        n1.visualize(filename=fn, rankdir="LR")
        with open(fn) as f:
            text = f.read()

        for word in ['rankdir', 'source1', 'source2', 'zip', 'map', 'add',
                     'shape=box', 'shape=ellipse']:
            assert word in text
```
```python
/home/christopher/mc/envs/dp_dev/bin/python /home/christopher/pycharm-2016.3/helpers/pycharm/_jb_pytest_runner.py --target test_graph.py::test_create_file
Testing started at 12:43 PM ...
Launching py.test with arguments test_graph.py::test_create_file in /home/christopher/dev/streamz/streamz/tests

============================= test session starts ==============================
platform linux -- Python 3.5.4, pytest-3.2.2, py-1.4.34, pluggy-0.4.0
rootdir: /home/christopher/dev/streamz, inifile:
plugins: xonsh-0.5.12, env-0.6.0
collected 1 item
test_graph.py .Error: Unknown HTML element <lambda> on line 1 
in label of node map; <lambda>
Error: Unknown HTML element <lambda> on line 1 
in label of node map; <lambda>
Error: Unknown HTML element <lambda> on line 1 
in label of node map; <lambda>
```
I'm following up on this, I'm not certain how this broke from our previous versions.

**Comment by CJ-Wright** [comment]
Also it seems that we could use a more robust assert than just checking the file's existence.

**Comment by ordirules** [comment]
having same problem:
```python
In [1]: from streamz import Stream

In [2]: s = Stream()

In [3]: s2 = s.map(lambda x : x + 1)

In [4]: s.visualize()
Error: Unknown HTML element <lambda> on line 1 
in label of node map; <lambda>
---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)
```

but this works:
```python
In [1]: from streamz import Stream

In [2]: s =Stream()

In [3]: def inc(x):
   ...:     return x+1
   ...: 

In [4]: s2 =s.map(inc)

In [5]: s.visualize()
Out[5]: <IPython.core.display.Image object>

```

**Comment by jrmlhermitte** [comment]
Should be closed with #81. @CJ-Wright let us know otherwise. thanks for the input!
